bundled-with-dotnet-single-file-deployment.yml

```diff
@@ -0,0 +1,20 @@
+rule:
+  meta:
+    name: bundled with .NET single-file deployment
+    namespace: runtime/dotnet
```

Comment:
should this go into the `internal\limitation\file` directory/namespace to avoid further processing?